build.gradle.kts

```diff
@@ -43,7 +43,9 @@ allprojects {
 dependencies {
     implementation("io.netty:netty-all:$nettyVersion")
 
+    testImplementation("org.assertj:assertj-core:$assertjVersion")
```

Comment:
Move it one line down. I prefer kotlin to be first and the rest in alphabetical order.